lib/vector-core/src/transforms/lua/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod v2;
```

Comment:
I am surprised to see this here. What is the logic behind splitting this transform up?

Reply:
I don’t love it. The problem are the ‘From’ implementations. They have to be in the same crate that defines the type. 

I’m open to suggestions. It’s not a pattern I’d like to see regularly.